### DISCUSSION TITLE
Is there a version of CC for 1.19.3 in which the peripherals actually work?

### DISCUSSION THREAD

**Discussion by fatboychummy** [cc-tweaked/CC-Tweaked#1533] [Q&A] [answered]
Heyo, started a pack on 1.19.3 since there was a mod I wanted to try that was only available for that version. I ended up not liking that mod, but I have a neat world I'd love to continue.

Unfortunately, I'm running into a spot now where I need to write some stuff for automating some things and unfortunately no matter what I do I cannot get peripherals to attach via a wired modem.

Peripherals seem to work fine if they're directly attached to the computer, but via a modem? Nope. Nonexistent.

![image](https://github.com/cc-tweaked/CC-Tweaked/assets/39929480/e4a945ab-719a-40c0-b129-569b109011f3)

![image](https://github.com/cc-tweaked/CC-Tweaked/assets/39929480/d64df778-aa64-4ce3-891e-506be0ca19db)

I'll also note that computers seem to not recognize disk drives... At all. At least. One computer can edit the files on the disk drive, but as soon as another computer tries to mount the drive the entire drive seems to... Fail?

Thus, I was wondering if there'd be a way to fix this. Perhaps by rolling back (if a rollback would even work?), or by applying some commit to the 1.19.3 version that would fix this? If that has a chance of working, I have no clue where to start unfortunately :(

I would make this an issue, but I don't think CC:T is actively dev'd for 1.19.3 anymore since it's a weird "in-between" version.

**Comment by SquidDev** [comment, marked as answer]
> I ended up not liking that mod, but I have a neat world I'd love to continue.

You should be able to update to 1.19.4 without problem. That's going to be your best bet - 1.19.3 was a bit of a weird in-between version where we were still sorting out a lot of issues with the multi-loader rework.

**Reply by fatboychummy** [reply]
Oh! Everything I googled stated that I couldn't update to 1.19.4, something about the worlds being too different or something.

Well, if that's the case then the fix is just to go ahead and do that. I'll give it a try when I get to my PC today.

Thanks!

**Reply by Lupus590** [reply]
Make a backup so that if stuff does break then you can still go back.
